lib/TikZImage.pm

```diff
@@ -2,7 +2,6 @@
 ################################################################################
 # WeBWorK Online Homework Delivery System
 # Copyright &copy; 2000-2018 The WeBWorK Project, http://openwebwork.sf.net/
-# $CVSHeader: pg/macros/parserMultiAnswer.pl,v 1.11 2009/06/25 23:28:44 gage Exp $
```

Comment:
Thanks.  These were cut and paste errors on my part.  You did the same thing I did, and searched the code for referenece to MultiAnswer, and found these!